frontend/actions.js

```diff
@@ -155,6 +171,7 @@ export default ({setState, getState}) => {
     ) {
       setState({
         transactionFee,
+        sendAmountForTransactionFee: amount,
```

Comment:
I see this value is not guaranteed to be an `int` -- doesn't something downstream break if `sendAmount.fieldValue` is a bit nasty (though I am not sure such nasty value passing validation readily exists)?

Reply:
fixed - amount is now converted to int explicitly to Int above